recipe/gen_patch_json.py

```diff
@@ -498,6 +498,9 @@ def _gen_new_index(repodata, subdir):
         if any(dep.startswith("gf2x") for dep in deps):
             _pin_stricter(fn, record, "gf2x", "x.x")
 
+        if any(dep.startswith("libnetcdf >=4.7.3") for dep in deps):
```

Comment:
@ocefpaf, is it okay to fix only packages built with `4.7.3`?

Reply:
Maybe >=4.7.0 to be safe? B/c old build with 4.7.{0,1,2} will install 4.7.4 too, right?

Reply:
What do you think? Does that makes sense?

Reply:
There are no builds with 4.7.0, 4.7.2. `4.7.1` is pinned to `x.x.x`

Reply:
Great. So let's get this one merge then!